script/update-statuses-project.rb

```diff
@@ -11,7 +11,7 @@
 class QueryExecutionError < StandardError; end
 NOTE_SEPARATOR = " --- "
 
-module Github
+module GitHub
```

Comment:
Often projects don't like renaming modules. So I expect I'll drop this commit (I didn't include it for another repository).

Reply:
Actually I think renaming in this case is a good idea, since it is indeed spelled GitHub with the two capitals. We just need to make sure the code still works 😅

Reply:
Ok. At some point I'll have to figure out which was the other repo w/ this and also offer it there...

Reply:
We maintain a number of projects, but maybe you're thinking of github/view_component?

Reply:
https://github.com/primer/react/blob/cfbafe63e00a863d1940f8b3671bf5a21ef2691b/script/component-status-project/deploy.rb#L15

I didn't include that change in primer/react#1905

Reply:
Hmm ok, interesting. Obviously this code isn't shared, but it's good to be consistent. Can I ask that you update primer/react to use the properly cased `GitHub`?